main/src/main/resources/org/clulab/numeric/WEEK.tsv

```diff
@@ -5,12 +5,24 @@
 # note: multi-word phrases must be tokenized in the same way as our tokenizer. If not sure, try the phrases in ./shell first!
 #
 first week       // XXXX-XX-01 -- XXXX-XX-07
-1st week       // XXXX-XX-01 -- XXXX-XX-07
+1st week         // XXXX-XX-01 -- XXXX-XX-07
```

Comment:
I'm not sure this is what it really means, but that might have been discussed at length previously.  If Saturday is the 1st, then the first week might be 3-9 for me.  Haven't I seen it written -W1?  The first week of June is XXXX-06-W1.

Reply:
I will work on #778 and tidy a few things below and merge it all at once if that's OK.